2022q1/kde.adoc

```diff
@@ -11,16 +11,38 @@ The software includes a full desktop environment called KDE Plasma (for both X11
 
 The KDE team (kde@) is part of desktop@ and x11@ as well, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphics-based desktop machine.
 
-*KDE Qt Patch Collection* The Qt Company stopped releasing Qt 5.15 updates under Open Source licenses in 2020. Qt 6 is now released under an Open Source license, but for the world of Open Source software that requires Qt 5, there is still a need for updates. The KDE Community fills that need by maintaining a curated set of patches -- generally backported from Qt6 -- to maintain the Open Source version of Qt 5. FreeBSD ports now use this *KDE Qt Patch Collection*, rather than the outdated last Qt 5.15.2 release from the Qt Company.
+*KDE Qt Patch Collection* The Qt Company did not release Qt 5.15 updates under Open Source licenses in 2020, leaving the Open Source 5.15 version lagging behind the proprietary release. Qt 6 is released under an Open Source license, but for the world of Open Source software that requires Qt 5, there is still a need for updates. The KDE Community fills that need by maintaining a curated set of patches -- generally backported from Qt6 -- to maintain the Open Source version of Qt 5. FreeBSD ports now use this *KDE Qt Patch Collection*, rather than the outdated last Qt 5.15.2 release from the Qt Company.
 This landed both in main and the last quarterly branch for 2021, since it brings important bugfixes.
 
 ==== KDE Stack
 
-* *KDE Plasma Desktop* (all the `*/plasma5-*` ports) was updated to 5.23.5 at the start of the year. Since this happened very shortly after quarterly was branched, this was MFH'ed.
-* *KDE Gear* (the collection of KDE libraries and applicatious outside of the Frameworks and Plasma Desktop groups) was updated to 21.12.1 and MFH'ed.
+* *KDE Plasma Desktop* (all the `*/plasma5-*` ports) was updated to 5.23.5 at the start of the year. Since this happened very shortly after quarterly was branched, this was MFH'ed. The long-term-support release 5.24 landed mid-february. The FreeBSD ports do not stick to LTS releases, and will follow the regular release schedule. 5.24.3 landed on schedule in March.
+* *KDE Gear* (the collection of KDE libraries and applicatious outside of the Frameworks and Plasma Desktop groups) was updated to 21.12.1 and MFH'ed. Monthy releases landed as well: 21.12.2 in February.
+* *KDE Frameworks* The frameworks have a monthly release cadence, so 5.90 landed in January, and 5.91 in February and 5.92 in March.
+* *KDE PIM* Because Google has changed the available REST API, KDE PIM currently does not support Contacts stored in a Google account.
```

Comment:
I'd swap clauses and remove the 2nd "KDE PIM": "*KDE PIM* currently does not support Contacts stored in a Google account because Google has changed the available REST API."